docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`-` Decimals now defaults to 18 when token contract info query fails.
```

Comment:
I have said this a thousand times but **changelog should be user oriented**

The user does not know what a token contract info query is or when it may fail. This is how you write it.

```suggestion
* :bug:`-` Decimals for a token will now default to 18 if no value can be found.
```